custom_components/tesla_custom/strings.json

```diff
@@ -12,8 +12,12 @@
     },
     "step": {
       "user": {
-        "description": "The next step will forward to the proxy for Tesla authentication. Tesla protects the login with a web application firewall (WAF) which may require a refresh to get or receive data. Please be cautious with multiple refreshes in a short period of time.",
-        "title": "Tesla - Configuration"
+        "description": "Use 'Auth App for Tesla' on iOS or 'Tesla Tokens' on Android\r\n to create a refresh token and enter it below.",
```

Comment:
I believe this is Markdown, do links work?

Reply:
Did not think to try markdown. I tried html and it borks.